examples/apps/cloudfront-ab-test/template.yaml

```diff
@@ -1,6 +1,11 @@
 AWSTemplateFormatVersion: '2010-09-09'
 Transform: 'AWS::Serverless-2016-10-31'
 Description: 'Blueprint for CloudFront ab testing, implemented in NodeJS.'
+Metadata:
+  Name: cloudfrontabtest
```

Comment:
Remove this metadata. It'll actually cause publish to fail because the published name (cloudfront-ab-test) does not match this field. Should just remove the metadata completely.